packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -49,27 +51,73 @@ const metadata: Meta<MenuButtonArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description:
+                'The text content of the button. This will be hidden when `content-hidden` is set but should always be provided; see [Accessibility](./?path=/docs/components-menu-button--docs#accessibility) for more info.',
+            table: { category: argsTableCategory.slots }
+        },
+        icon: {
+            name: 'start',
+            description: iconDescription,
+            table: { category: argsTableCategory.slots }
+        },
+        endIcon: {
+            name: 'end',
+            description: endIconDescription,
+            table: { category: argsTableCategory.slots }
+        },
+        menu: {
+            description:
+                'The [nimble-menu](./?path=/docs/components-menu--docs) to be displayed when the button is toggled.',
```

Comment:
this link is working well for me in the chromactic.com published storybook in FFX and Chrome 👍